apps/meeting/scripts/utilities.js

```diff
@@ -0,0 +1,36 @@
+const exec = require('child_process').spawnSync;
+
+const logger = {
+  error: (output, info = '') =>
```

Comment:
Remove 2nd argument since it's not being used? Otherwise, you can make all three log methods fully extnesible by taking `...args`